example/properties.yml

```diff
@@ -18,3 +18,23 @@ api_server_properties:
     min_connections: 0
   scheduler_uri: https://10.244.4.4:8080
   port: 3002
+
+cf_properties:
+  api: https://api.bosh-lite.com
+  grant_type: password
+  username: admin
+  password: admin
+  client_id: admin
+  secret: admin-secret
+  
+metricscollector_properties:
```

Comment:
Some of these should be pushed into the spec as defaults.
8080 is a bad port to use as a default.